src/com/epam/andrei_eremenko/task_01/module/birthday/GroupCreator.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.andrei_eremenko.task_01.module.birthday;
+
+import java.util.Random;
+
+class GroupCreator {
+
+    static void createGroups(short[][] array) {
+        Random rand = new Random();
```

Comment:
Save and re-use this Random, move it to field
`static Random rand = new Random();`